website/source/api/agent.html.md

```diff
@@ -0,0 +1,468 @@
+      "commit_index": "144",
+      "term": "2",
+      "last_log_index": "144",
+      "protocol_version_max": "3",
+      "snapshot_version_max": "1",
+      "latest_configuration_index": "1",
+      "latest_configuration": "[{Suffrage:Voter ID:127.0.0.1:4647 Address:127.0.0.1:4647}]",
+      "last_contact": "never",
+      "applied_index": "144",
+      "protocol_version": "1",
+      "protocol_version_min": "0",
+      "snapshot_version_min": "0",
+      "state": "Leader",
+      "last_snapshot_term": "0"
+    },
+    "client": {
+      "heartbeat_ttl": "17.79568937s",
+      "node_id": "fb2170a8-257d-3c64-b14d-bc06cc94e34c",
+      "known_servers": "127.0.0.1:4647",
+      "num_allocations": "0",
+      "last_heartbeat": "10.107423052s"
+    },
+    "serf": {
+      "event_time": "1",
+      "event_queue": "0",
+      "encrypted": "false",
+      "member_time": "1",
+      "query_time": "1",
+      "intent_queue": "0",
+      "query_queue": "0",
+      "members": "1",
+      "failed": "0",
+      "left": "0",
+      "health_score": "0"
+    }
+  }
+}
+```
+
+## Join Agent
+
+This endpoint introduces a new member to the gossip pool. This endpoint is only
+eligible for servers.
+
+| Method | Path                         | Produces                   |
+| ------ | ---------------------------- | -------------------------- |
+| `POST` | `/agent/join`                | `application/json`         |
+
+The table below shows this endpoint's support for
+[blocking queries](/api/index.html#blocking-queries),
+[consistency modes](/api/index.html#consistency-modes), and
+[required ACLs](/api/index.html#acls).
+
+| Blocking Queries | Consistency Modes | ACL Required |
+| ---------------- | ----------------- | ------------ |
+| `NO`             | `none`            | `none`       |
+
+### Parameters
+
+- `address` `(string: <required>)` - Specifies the address to join in the
+  `ip:port` format. This is provided as a query parameter and may be specified
+  multiple times to join multiple servers.
+
+### Sample Request
+
+```text
+$ curl \
+    --request POST \
+    https://nomad.rocks/v1/agent/join?address=1.2.3.4&address=5.6.7.8
+```
+
+### Sample Response
+
+```json
+{
+  "error": "",
+  "num_joined": 2
+}
+```
+
+## Force Leave Agent
+
+This endpoint forces a member of the gossip pool from the `"failed"` state to
+the `"left"` state. This allows the consensus protocol to remove the peer and
+stop attempting replication. This is only applicable for servers.
+
+| Method | Path                         | Produces                   |
+| ------ | ---------------------------- | -------------------------- |
+| `POST` | `/agent/force-leave`         | `application/json`         |
+
+The table below shows this endpoint's support for
+[blocking queries](/api/index.html#blocking-queries),
+[consistency modes](/api/index.html#consistency-modes), and
+[required ACLs](/api/index.html#acls).
+
+| Blocking Queries | Consistency Modes | ACL Required |
```

Comment:
Hey @dadgar 

Thank you for the comment. I was aiming for consistency between Nomad and Consul's docs as much as possible. Do you think having this section at all makes sense for Nomad?